flink-runtime/src/main/java/org/apache/flink/streaming/api/environment/CheckpointConfig.java

```diff
@@ -623,8 +623,8 @@ public void configure(ReadableConfig configuration) {
                 .getOptional(CheckpointingOptions.ENABLE_UNALIGNED)
                 .ifPresent(this::enableUnalignedCheckpoints);
         configuration
-                .getOptional(CheckpointingOptions.ENABLE_UNALIGNED_INTERRUPTIBLE_TIMERS)
-                .ifPresent(this::enableUnalignedCheckpointsInterruptibleTimers);
+                .getOptional(CheckpointingOptions.ENABLE_UNALIGNED_SPLITTABLE_TIMERS)
+                .ifPresent(this::enableUnalignedCheckpointsSplitttableTimers);
```

Comment:
nit: enableUnalignedCheckpointsSplitttableTimers has 3 ts here - typo?

Reply:
ha yes, I dont want to introduce another typo while trying to fix a typo